integration_tests/__tests__/__snapshots__/failures-test.js.snap

```diff
@@ -103,3 +103,272 @@ Ran all test suites matching \\"assertion-count-test.js\\".
+    -       \\"c\\": 6,
+    +       \\"c\\": 5,
+          },
+        },
+      }
+      
+      at Object.<anonymous> (__tests__/node-assertion-error-test.js:40:10)
+
+  ● assert.deepEqual with a message
+
+    assert.deepEqual(received, expected)
+    
+    Expected value to deepEqual to:
+      {\\"a\\": {\\"b\\": {\\"c\\": 7}}}
+    Received:
+      {\\"a\\": {\\"b\\": {\\"c\\": 5}}}
+    
+    Message:
+      this is a message
+    
+    Difference:
+    
+    - Expected
+    + Received
+    
+    Object {
+        \\"a\\": Object {
+          \\"b\\": Object {
+    -       \\"c\\": 7,
+    +       \\"c\\": 5,
+          },
+        },
+      }
+      
+      at Object.<anonymous> (__tests__/node-assertion-error-test.js:44:10)
+
+  ● assert.notDeepEqual
+
+    assert.notDeepEqual(received, expected)
+    
+    Expected value to notDeepEqual to:
+      {\\"a\\": 1}
+    Received:
+      {\\"a\\": 1}
+    
+    Difference:
+    
+    Compared values have no visual difference.
+      
+      at Object.<anonymous> (__tests__/node-assertion-error-test.js:48:10)
+
+  ● assert.strictEqual
+
+    assert.strictEqual(received, expected)
+    
+    Expected value to be (operator: ===):
+      NaN
+    Received:
+      1
+      
+      at Object.<anonymous> (__tests__/node-assertion-error-test.js:52:10)
+
+  ● assert.notStrictEqual
+
+    assert.notStrictEqual(received, expected)
+    
+    Expected value not to be (operator: !==):
+      1
+    Received:
+      1
+    
+    Message:
+      My custom error message
+    
+    Difference:
+    
+    Compared values have no visual difference.
+      
+      at Object.<anonymous> (__tests__/node-assertion-error-test.js:56:10)
+
+  ● assert.deepStrictEqual
+
+    assert.deepStrictEqual(received, expected)
+    
+    Expected value to deepStrictEqual to:
+      {\\"a\\": 2}
+    Received:
+      {\\"a\\": 1}
+    
+    Difference:
+    
+    - Expected
+    + Received
+    
+    Object {
+    -   \\"a\\": 2,
+    +   \\"a\\": 1,
+      }
+      
+      at Object.<anonymous> (__tests__/node-assertion-error-test.js:60:10)
+
+  ● assert.notDeepStrictEqual
+
+    assert.notDeepStrictEqual(received, expected)
+    
+    Expected value to notDeepStrictEqual to:
+      {\\"a\\": 1}
+    Received:
+      {\\"a\\": 1}
+    
+    Difference:
+    
+    Compared values have no visual difference.
+      
+      at Object.<anonymous> (__tests__/node-assertion-error-test.js:64:10)
+
+  ● assert.ifError
+
+    Error
+      1 thrown
+
+  ● assert.doesNotThrow
+
+    assert.doesNotThrow(function)
+    
+    Expected the function not to throw an error.
+    Instead, it threw:
+      [Error: err!]
+    
+    Message:
+      Got unwanted exception..
+      
+      at Object.<anonymous> (__tests__/node-assertion-error-test.js:72:10)
+
+  ● assert.throws
+
+    assert.throws(function)
+    
+    Expected the function to throw an error.
+    But it didn't throw anything.
+    
+    Message:
+      Missing expected exception..
```

Comment:
Yes, `assert` generates this message and marks that it's not generated 🤷‍♂️, that's why it's shown here.